src/view/store.ts

```diff
@@ -47,22 +49,17 @@ export const useGetDiceStore = () => useGetStore<Record<string, Node>, IDiceActi
   },
 
   // Sync view state with Fluid state by loading default state or patching the key that changed
-  reducer: (model, state, { type, changed }) => {
-    let newState;
+  reducer: (model, draft, { type, changed }) => {
```

Comment:
Could we keep the naming here as "state"? Since draft is an immer specific concept, I'm wondering if we could just abstract the usage of immer away (and its nicely done here already since its all behind the `useGetStore` function). If we keep the name here as "state", I think it'll be easy for devs to just interpret that as the same object type as what is established in `initialState`
```suggestion
  reducer: (model, state, { type, changed }) => {
```

Reply:
I went with draft so that users know that it is NOT state, and therefore you CAN modify it directly.